website/source/guides/identity/approle-trusted-entities.html.md

```diff
@@ -0,0 +1,779 @@
+└── variables.tf
+```
+
+**Task 2:** Update the `terraform.tfvars.example` file to match your account and
+rename it to `terraform.tfvars`.
+
+At minimum, replace the following variable with appropriate values:
+
+- **`s3_bucket_name`**
+- **`vpc_id`**
+- **`subnet_id`**
+- **`key_name`**
+- **`ec2_pem`**
+
+> NOTE: If your VPC, subnet and EC2 key pair were created on a region other than
+`us-east-1`, be sure to set the **`aws_region`** value to match your chosen region.
+
+**Task 3:** Perform a `terraform init` to pull down the necessary provider resources.
+Then `terraform plan` to verify your changes and the resources that will be
+created. If all looks good, then perform a `terraform apply` to provision the
+resources. The Terraform output will display the public IP address to SSH into
+your server.
+
+```plaintext
+$ terraform init
+Initializing provider plugins...
+...
+Terraform has been successfully initialized!
+
+
+$ terraform plan
+...
+Plan: 5 to add, 0 to change, 0 to destroy.
+
+
+$ terraform apply
+...
+Apply complete! Resources: 5 added, 0 changed, 0 destroyed.
+
+Outputs:
+vault-public-ip = 192.0.2.0
+```
+
+The Terraform output will display the public IP address to SSH into
+your server.
+
+**Task 4:** Initial setup of the Chef server takes several minutes. Once you can
+SSH into your mgmt server, run `tail -f /var/log/tf-user-data.log` to see when
+the initial configuration is complete.
+
+```plaintext
+$ tail -f /var/log/tf-user-data.log
+```
+
+When you see the following message, the initial setup is complete.
+
+```plaintext
++ echo '2018/03/27 21:53:06 /var/lib/cloud/instance/scripts/part-001: Complete'
+```
+
+You can find the following subfolders in
+your home directory:
+
+- **`/home/ubuntu/vault-chef-approle-demo`**: root of our project
+- **`/home/ubuntu/vault-chef-approle-demo/chef`**: root of our Chef app; this is
+where our `knife` configuration is located (`.chef/knife.rb`)
+- **`/home/ubuntu/vault-chef-approle-demo/scripts`**: there's a
+`vault-approle-setup.sh` script located here to help automate the setup of
+Vault, or you can follow along in the rest of this README to configure Vault
+manually
+
+### Step 2: Initialize and Unseal Vault
+
+Before moving on, set your working environment variables in your mgmt server:
+
+```plaintext
+$ export VAULT_ADDR=http://127.0.0.1:8200
+$ export VAULT_SKIP_VERIFY=true
+```
+
+Before you can do anything in Vault, you need to initialize and unseal it.
+Perform ***one*** of the following:
+
+- **Option 1:** Run the `/home/ubuntu/demo_setup.sh` script to get up and running, and proceed to
+[Phase 2 - Provision our Chef Node to Show AppRole Login](#phase2).
+- **Option 2:** Continue onto [Step 3: AppRole Setup](#step-3-approle-setup) to
+set up the demo environment ***manually***.
+
+
+### Step 3: AppRole Setup
+
+First, initialize and unseal the Vault server using a shortcut.
+
+~> This is a convenient shortcut for demo. **_DO NOT DO THIS IN PRODUCTION!!!_**
```

Comment:
Maybe we can add a link to the docs for how to unseal using best practices so they can choose to do it the correct way if they want.